app/views/puppetclasses/_form.html.erb

```diff
@@ -45,11 +45,11 @@
           </div>
         </div>
         <div class="smart-var-left col-md-3">
-          <ul class="nav nav-pills nav-stacked smart-var-tabs" data-tabs="pills">
+          <ul class="nav nav-pills nav-stacked smart-var-tabs" data-tabs="pills">yeah
             <% @puppetclass.class_params.includes(:environments, :environment_classes, :lookup_values).each do |key| %>
               <li data-used-environments=<%= key.environments.map(&:to_s).to_json %> >
                 <a data-toggle="tab" id="pill_<%= key.to_param %>" href="#<%= key.to_param %>" title="<%= key %>">
-                  <div class="clip"><%= icon_text((key.override ? "flag": ""), key.to_s.tr('_',' ')) %></div>
+                  <div class="clip"><%= icon_text((key.override ? "flag": ""), key.to_s.tr('_',' '), :kind => 'fa') %></div>
```

Comment:
I didn't like the 'pficon-flag' icon, the 'fa-flag' is closer to what we have today.